docs/whats-new.rst

```diff
@@ -11,6 +11,9 @@ v0.3 (unreleased)
 Breaking Changes
 ~~~~~~~~~~~~~~~~
 
+- Deprecate ``Constant`` class, so now we have a ``_constants.py`` file
+  containing commonly used constants. (fixes :issue:`50` via :pull:`223`).
```

Comment:
nit: let's try and be a little more specific, something along the lines of "... so now we have a ``_constants.py`` module containing only the physical constants required by ``aospy``, stored as scalar ``DataArrays``."

Reply:
Re-word as "Deprecate ``Constant`` class and ``constants.py`` module.  Physical constants used internally by aospy are now stored in ``_constants.py``  (fixes :issue:`50` via :pull:`223`)."